CONTRIBUTING.md

```diff
@@ -45,34 +45,33 @@ When developing a new component, add it to the `packages/ember-toucan-core` dire
 1. Components are added under `packages/ember-toucan-core/src/components`
 2. When adding a new component, it needs to be added to the template registry so that Glint picks it up. To do that, add a line in `packages/ember-toucan-core/src/template-registry.ts`.
 
-## Running the docs/test-app
-
-Prior to running the `docs-app` or `test-app`, you'll need to install dependencies:
-
-```bash
-cd docs-app
-pnpm i
-pnpm start
-```
-
-```bash
-cd test-app
-pnpm i
-pnpm start
-```
-
 ## Viewing changes in the docs/test-app
 
 [We're working on making this better](https://github.com/CrowdStrike/ember-toucan-core/issues/40), but for the time being you can use `pnpm link` to create a symlink between the addon and the apps. This will allow you to make a change in the addon and immediately see the change in the app rather than having to stop the app, rebuild, resync dependencies, etc.
 
 ```bash
-# Create a symlink in the addon
+# Create a symlink in the ember-toucan-core addon
 cd packages/ember-toucan-core
+pnpm build
```

Comment:
Getting an error running `pnpm build` here:

```
----- [12:05:02 dwenzel ~/Opensource/ember-toucan-core/packages/ember-toucan-core (main)]
$ pnpm build

> @crowdstrike/ember-toucan-core@0.1.1 build /Users/dwenzel/Opensource/ember-toucan-core/packages/ember-toucan-core
> rollup --config

[!] SyntaxError: Unexpected identifier
    at Loader.moduleStrategy (internal/modules/esm/translators.js:145:18)


 ELIFECYCLE  Command failed with exit code 1.
 ```

Reply:
I haven't seen that error message before when I get a build failure.

My `pnpm --version` is `7.27`, I believe Tony is on `7.30.0`

Reply:
Maybe @ynotdraw has seen this error before?

Reply:
or @simonihmig ... apologies about pinging so many busy folks here, but I am stumped on that error ^^.

Reply:
So I tried this with my new laptop, and I didn't get the error.  :shrug: